brands/shop/variety_store.json

```diff
@@ -51,6 +51,17 @@
       "shop": "variety_store"
     }
   },
+  "shop/variety_store|Daiso Japan": {
+    "countryCodes": ["au"],
+    "matchNames": ["Daiso"],
```

Comment:
You can remove the matchName here since it is the same as the `short_name`. 
(we automatically match `alt_name`, `short_name` and `official_name` tags, see #2732)

Reply:
I think that this kind of magic may be included in a build script.

Reply:
done in https://github.com/osmlab/name-suggestion-index/pull/2772

Reply:
Thanks I didn't realise short_name was included.